### ISSUE TITLE
`# credo:disable-for-next-line` not working

### ISSUE THREAD

**Issue by nathanl** [rrrene/credo#1177]
I just updated to credo 1.7.11 from 1.7.8 and got a new Credo warning: ` Function takes too many parameters (arity is 9, max is 8).`

The function already had `credo:disable-for-next-line` on the line before it.

```
  # credo:disable-for-next-line
  defp some_func(arg1, arg2, _, arg4, arg5, arg6, arg7, arg7, arg9)
    :do_stuff
  end
```

If I put `# credo:disable-for-this-file` at the top of the file, it disables the warning.

**Comment by rrrene** [comment]
@nathanl I wasn't able to reproduce this either. Mmmh, any other info you can share?

**Comment by rrrene** [comment]
Any new input on this?

**Comment by rrrene** [comment]
I apologize for the age/inactivity on this issue. I should have done a better job at following up properly. 😥 

Please feel free to re-open this issue at your discretion.
